Khan/wonder-blocks
# Add inline variant to Link

## Summary:
Add the `inline` variant to Link. Make it match the design spec
https://www.figma.com/file/VbVu3h2BpBhH80niq101MHHE/%F0%9F%92%A0-Main-Components?node-id=11512%3A7203&t=eFPXHDHuxnSBGKzI-0

Issue: https://khanacademy.atlassian.net/browse/WB-1450

## Test plan:
`yarn jest`

Storybook interaction tests
https://5e1bf4b385e3fb0020b7073c-wqfijoyjxy.chromatic.com/?path=/story/link--inline
https://5e1bf4b385e3fb0020b7073c-wqfijoyjxy.chromatic.com/?path=/story/link--inline-light

Storybook manual testing
- Check colors of resting links
- Check colors of hovered links
- Check colors of mousdown links
- Check outline of focused links
- Add focus to link and hover at the same time. The focus ring should stay.
